### PR TITLE
Add rsql

### PR DESCRIPTION
**Pull request by DanilBut**

### CODE REVIEW

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/dao/rsql/RsqlSpecification.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RsqlSpecification<T> implements Specification<T> {
+
+    private String property;
+    private ComparisonOperator operator;
+    private List<String> arguments;
+
+    public RsqlSpecification(final String property, final ComparisonOperator operator, final List<String> arguments) {
```
Comment (akarzhaneuski) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/dao/rsql/RsqlSpecification.java:19]: We already have RSQL configurations in master. Please merge it locally and remove duplicated code

File: sandbox-backend/build.gradle [hunk 2]
```diff
@@ -14,6 +14,9 @@ jar {
 
 
 dependencies {
+    implementation group: 'org.modelmapper', name: 'modelmapper'
```
Comment (akarzhaneuski) [inline sandbox-backend/build.gradle:17]: We already have all of this dependencies

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/web/CreateStatusDTO.java [hunk 3]
```diff
@@ -1,18 +0,0 @@
-package com.exadel.team2.sandbox.web;
```
Comment (akarzhaneuski) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/web/CreateStatusDTO.java:1]: A lot of removed classes, please double check and confirm that all removes are expected

**Review by akarzhaneuski — CHANGES_REQUESTED** [review]

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/dao/rsql/RsqlSpecification.java (hunk 1 above)
Comment (DanilBut) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/dao/rsql/RsqlSpecification.java]: Done.

File: sandbox-backend/build.gradle (hunk 2 above)
Comment (DanilBut) [reply sandbox-backend/build.gradle]: Removed

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/web/CreateStatusDTO.java (hunk 3 above)
Comment (DanilBut) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/web/CreateStatusDTO.java:1]: These dto classes were repetitive

**Pull request merged**